Formula/r/rapidyaml.rb

```diff
@@ -0,0 +1,27 @@
+class Rapidyaml < Formula
+  desc "Library to parse and emit YAML, and do it fast"
+  homepage "https://github.com/biojppm/rapidyaml"
+  url "https://github.com/biojppm/rapidyaml/releases/download/v0.7.2/rapidyaml-0.7.2-src.tgz"
+  sha256 "175b71074005a7d48ae03e973f1a7a5c104c588f193ba85044c5b4a97341aae0"
+
+  depends_on "cmake" => :build
+
+  conflicts_with "c4core", because: "both install `c4core`"
```

Comment:
also I did not see the 4core utility installed.

Reply:
The rapidyaml also install a ext `c4core` which conflict with c4core.
like `/opt/homebrew/include/c4` .